roles/openshift-applier/filter_plugins/applier-filters.py

```diff
@@ -41,12 +43,36 @@ def filter_applier_items(applier_list, filter_tags):
 
     return applier_list
 
+def check_file_location(path, tmp_inv_dir):
+    return_vals = {
+        "oc_option_f":  '',
+        "oc_file_path": path,
+        "oc_process_local": ''
+    }
+
+    file_status = os.path.isfile("%s/%s" % (tmp_inv_dir,path))
```

Comment:
Please remove the `/` part of this as otherwise if the `tmp_inv_dir` is set to nothing the returned path will start with `/` and that's potentially the wrong location of the file.